### PR TITLE
Balder/nap insteadof beautysleep

### PR DESCRIPTION
**Pull request by baldersheim**
@jonmv or @bjorncs PR

### CODE REVIEW

File: vespa-feed-client/src/main/java/ai/vespa/feed/client/impl/JettyCluster.java [hunk 1]
```diff
@@ -239,11 +239,7 @@ private static String endpointUri(URI uri) {
         return String.format("%s://%s:%s", uri.getScheme(), uri.getHost(), portOf(uri));
     }
 
-    private static class JettyResponse implements HttpResponse {
-        final Response response;
-        final byte[] content;
-
-        JettyResponse(Response response, byte[] content) { this.response = response; this.content = content; }
+    private record JettyResponse(Response response, byte[] content) implements HttpResponse {
```
Comment (bjorncs) [inline vespa-feed-client/src/main/java/ai/vespa/feed/client/impl/JettyCluster.java:242]: Records are not available on JDK11 - the version used by our internal Hadoop integration.

**Review by bjorncs — CHANGES_REQUESTED** [review]

File: vespa-feed-client/src/main/java/ai/vespa/feed/client/impl/JettyCluster.java (hunk 1 above)
Comment (baldersheim) [reply vespa-feed-client/src/main/java/ai/vespa/feed/client/impl/JettyCluster.java]: Reverted, but shouldn't this be limited in the pom file so intellij would not suggest it ?

Comment (baldersheim) [reply vespa-feed-client/src/main/java/ai/vespa/feed/client/impl/JettyCluster.java]: Did not fail maven build either....

**Review by bjorncs — APPROVED** [review]

File: vespa-feed-client/src/main/java/ai/vespa/feed/client/impl/JettyCluster.java (hunk 1 above)
Comment (bjorncs) [reply vespa-feed-client/src/main/java/ai/vespa/feed/client/impl/JettyCluster.java]: That is unexpected. The build file targets 11 for all clients.

**Comment by jonmv** [comment]
👍

**Pull request merged**